source/write-operations.txt

```diff
@@ -0,0 +1,185 @@
+.. _kotlin-sync-write:
+
+=====================
+Write Data to MongoDB
+=====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :description: Learn how to use the Kotlin Sync driver to write data to MongoDB.
+   :keywords: usage examples, save, crud, create, code example
+
+.. .. toctree::
+..    :titlesonly:
+..    :maxdepth: 1
+.. 
+..    /write/insert
+..    /write/update
+..    /write/replace
+..    /write/delete
+..    /write/bulk-write
+..    /write/gridfs
+
+Overview
+--------
+
+On this page, you can see copyable code examples of common
+methods that you can use to write data to MongoDB by using the
+{+driver-short+}.
+
+.. tip::
+
+  To learn more about any of the methods shown on this page, see the link
+  provided in each section.
+
+To use an example from this page, copy the code example into the
+:ref:`sample application <kotlin-sync-write-sample>` or your own application.
+Be sure to replace all placeholders in the code examples, such as ``<connection string URI>``, with
+the relevant values for your MongoDB deployment.
+
+.. _kotlin-sync-write-sample:
+
+.. include:: /includes/usage-examples/sample-app-intro.rst
+
+.. literalinclude:: /includes/usage-examples/sample-app.kt
+   :language: kotlin
+   :copyable:
+   :linenos:
+   :emphasize-lines: 20-22
+
+Insert One
+----------
+
+The following code shows how to insert a single document into a
+collection:
+
+.. literalinclude:: /includes/usage-examples/write-code-examples.kt
+   :start-after: start-insert-one
+   :end-before: end-insert-one
+   :language: kotlin
+   :copyable:
+   :dedent:
+
+.. To learn more about the ``insert_one()`` method, see the :ref:`Insert Documents
+.. <kotlin-sync-write-insert>` guide.
+
+Insert Multiple
+---------------
+
+The following code shows how to insert multiple documents into a
+collection:
+
+.. literalinclude:: /includes/usage-examples/write-code-examples.kt
+   :start-after: start-insert-multiple
+   :end-before: end-insert-multiple
+   :language: kotlin
+   :copyable:
+   :dedent:
+
+.. To learn more about the ``insert_many()`` method, see the :ref:`Insert Documents
+.. <kotlin-sync-write-insert>` guide.
+
+Update One
+----------
+
+The following code shows how to update a single document in a
+collection by creating or renaming a field:
```

Comment:
Do we also want to mention reassigning the value of a field as an option here?

Reply:
updated language